README.md

```diff
@@ -1,2 +1,33 @@
-# chef-bluematador
-Chef cookbook to install the Blue Matador agent.
+# bluematador_agent
+
+Installs/Configures the Blue Matador Agent.
+
+## Requirements
+
+ * chef >= 12.1
+
+## Platforms
```

Comment:
we support windows. chef supports windows. why doesn't our chef support windows?